### PR TITLE
Parse DB parameters from .env.local

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
This is now working, however you'll need to comment out the rabbitmq commands and only use it to load the database, not elasticsearch as those components are not working at this time. This file shouldn't need to change when they are working later though.

### CODE REVIEW

File: bin/loaddump [hunk 1]
```diff
@@ -38,7 +38,27 @@ if [ ! -f $1 ]; then
     exit
 fi
 
-eval $(parse_yaml app/config/parameters.yml "")
+# Manually parse DATABASE_URL from .env.local file
+
+DATABASE_URL=$(grep "^DATABASE_URL" ./.env.local)
+IFS=':' read -ra ADDR <<< "$DATABASE_URL"
```
Comment (mickel1138) [inline bin/loaddump]: What is ADDR?

Comment (fightingtexasaggie) [reply bin/loaddump]: ok, I'll make less cryptic

**Comment by fightingtexasaggie** [comment]
> Getting an authentication error, using the same password

Make sure you use localhost and not 127.0.0.1 in the .env.local file.

**Comment by mickel1138** [comment]
@fightingtexasaggie We now have a field called database_name, you can use it if you like.

**Pull request merged**